### ISSUE TITLE
[Feature Request] Add option and methods to make input and button disabled

### ISSUE THREAD

**Issue by zaniar** [cocopon/tweakpane#215]
Hi!
Thank you for the excellent software.

In my use case, I need to make *input* and *button* disabled. Currently I made them disabled by hacking the DOM.
It would be great if **tweakpane** has:

1. `disabled: boolean` **or** `enabled: boolean` option for *input* and *button*
2. `enable()` and `disable()` methods **or** `setEnabled(toggle: boolean)` method for *input* and *button*

Keep up the good work!

**Comment by cocopon** [comment]
Added `disabled` parameter and `disabled` property to inputs and buttons.

```js
const pane = new Tweakpane();
const i = pane.addInput(PARAMS, 'param', {
  disabled: true,
  title: 'Advanced',
});

// ...

btn.on('click', () => {
  i.disabled = !i.disabled;
});
```

Please try the beta version `2.2.0-beta.0`.

**Comment by cocopon** [comment]
Released in 2.2.0.
